### DISCUSSION TITLE
fastlane-plugin-match

### DISCUSSION THREAD

**Discussion by chenguichun** [fastlane/fastlane#17450] [Q&A] [answered]
When I execute build install, I get an error, ask for answers, thanks
The error message is as follows:

<img width="852" alt="59ff0c80-9b6d-44ea-ae2e-340986a099a4" src="https://user-images.githubusercontent.com/15317493/96238977-af62c780-0fd1-11eb-9c60-8445df1cd236.png">

Could not find gem 'fastlane-plugin-match' in git@github.com:cgc/Apple_-Certification.git (at master@30e2d74).
The source does not contain any versions of 'fastlane-plugin-match'

**Comment by techinpark** [comment]
Hello! @chenguichun  can you share your `Fastfile`?

**Comment by rogerluan** [comment, marked as answer]
Duplicate of https://github.com/fastlane/fastlane/discussions/17462#discussioncomment-113528
